modules/administration-guide/partials/proc_using-minikube-to-set-up-kubernetes.adoc

```diff
@@ -14,8 +14,8 @@ This section describes how to use Minikube to prepare a local single-node {kuber
 
 .Procedure
 
-. Start Minikube (it is important to *allocate at least 4GB of RAM but 8GB are recommended*):
+. Start Minikube (it is important to allocate at least 8GB of RAM and 4 CPU cores):
 +
 ----
-$ minikube start --addons=ingress --vm=true --memory=8192
+$ minikube start --addons=ingress --vm=true --memory=8192 --cpus=4
```

Comment:
this is a min req I believe otherwise you will get insufficient CPU for gateway pods
![image](https://user-images.githubusercontent.com/1461122/164679784-3de3d561-f42c-4ac8-8024-13620c70602e.png)

Reply:
I can confirm. I experienced the same error.